### PR TITLE
feat(badge-patch): fix for badge text center

### PR DESCRIPTION
**Pull request by abhijeetj10**
1. patch fix for badge text center.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vonage/vivid-react",
-  "version": "2.10.92",
+  "version": "2.10.93",
```
Comment (dvelluto) [inline package.json:3]: did you generated this with the yarn command or manually pushed?

in this project usually version is updated with the yarn command which is also tagging

File: src/patches/vwcBadge.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**
+* Patches provided VwcBadge instance to show text in the center
```
Comment (dvelluto) [inline src/patches/vwcBadge.js:2]: specify that this is for V3 of badges to not  confuse with v2

File: package.json (hunk 1 above)
Comment (abhijeetj10) [reply package.json]: done

File: src/patches/vwcBadge.js (hunk 2 above)
Comment (abhijeetj10) [reply src/patches/vwcBadge.js]: done

**Review by dvelluto — APPROVED** [review]

**Pull request merged**